tests/helpers.py

```diff
@@ -10,6 +10,8 @@
 from tests.quirks import KNOWN_GYM_RENDER_NOT_IMPLEMENTED
 
 
+# pylint: disable=missing-param-doc, missing-type-doc
+# pylint: disable=missing-return-doc, missing-return-type-doc
```

Comment:
Please add docstring for this function. We usually will not disable pylint unless necessary. You can explore the codebase and look for examples.

Reply:
I have added docstring for most functions. However, there are several functions I need more time to understand its parameters and returns for example recurrent_step_lstm, recurrent_step_gru.

Reply:
@yeukfu @ahtsan should be able to help you fill those out.

Reply:
great